custom/icds_reports/tests/agg_tests/test_export_data.py

```diff
@@ -3813,7 +3813,7 @@ def test_growth_tracker_export(self):
                 [
                     'Export Info',
                     [
-                        ['Generated at', '16:21:11 15 November 2017'],
+                        ['Generated at', self.now],
```

Comment:
nit: there are other instances of this constant that could be replaced (lines: 69, 556, 656, 723, 819, 887, 983, 1017, 1332, 1436, 2392, 3900, 4022, 4133).

Reply:
thanks @millerdev